container_pipeline/workers/scanners/rpm_verify.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+
+import docker
+import json
+import logging
+import os
+import shutil
+import subprocess
+import config
+import hashlib
+
+from Atomic import Atomic, mount
+
+from container_pipeline.utils import Build, get_job_name
+from container_pipeline.lib.log import load_logger
+from container_pipeline.lib.settings import SCANNERS_OUTPUT
+
+class ScannerRPMVerify(object):
+    """
+    scanner-rpm-verify atomic scanner handler
+    """
+
+    def __init__(self):
+        self.scanner_name = "scanner-rpm-verify"
+        self.full_scanner_name = \
+            "registry.centos.org/pipeline-images/scanner-rpm-verify"
+        self.atomic_object = Atomic()
+
+        #Add logger
+        load_logger()
+        self.logger = logging.getlogger('scan-worker')
+
+    def run_atomic_scanner(self):
+        """
+        Run the atomic scan command
+        """
+        process = subprocess.Popen([
+            "atomic",
+            "scan",
+            "--scanner=rpm-verify",
+            "%s" % self.image_id
+        ], stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+
```

Comment:
In all other scanners, we are either creating a `cmd` or `scan_cmd` list, passing it to `scan` method in super-class, performing actions based on the returned results. This is inconsistent with that behavior.